docs/indepth/installation.rst

```diff
@@ -386,14 +386,6 @@ To enable and start Docker, run the following commands in a root console:
 .. task:: enable_docker
    :prompt: [root@node]#
 
-To enable Flocker to forward ports between nodes, the firewall needs to be configured to allow forwarding.
-On a typical fedora installation, the firewall is configured by `firewalld <https://fedoraproject.org/wiki/FirewallD>`_.
-(Note: The Fedora AWS images don't have firewalld installed, as there is an external firewall configuration.)
```

Comment:
Maybe we want to document that AWS has an external firewall? But I think that is out of the scope of this ticket.